channels/ae.m3u

```diff
@@ -67,6 +67,8 @@ https://dmitwlvvll.cdn.mangomolo.com/dubaisportshd5/smil:dubaisportshd5.smil/ind
 https://dmisxthvll.cdn.mgmlcdn.com/dubaitvht/smil:dubaitv.stream.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="DubaiZaman.ae" tvg-country="AE" tvg-language="Arabic" tvg-logo="https://i.imgur.com/F5U9qpf.png" group-title="Classic",Dubai Zaman (576p)
 https://dmiffthvll.cdn.mangomolo.com/dubaizaman/smil:dubaizaman.stream.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-country="AE" tvg-language="Arabic" tvg-logo="" group-title="",EXPO 2020
+https://shls-expotv-prod-dub.shahid.net/out/v1/d01b0b3888284878b8898017895a5922/index.m3u8
```

Comment:
From what I see, it's a stream that's not meant to stay all the time online, since it's an event.

Reply:
I think it will broadcast for a few months

Reply:
I see. I do not known if it's a good idea to add this one, knowing it's meant to disappear a few months later.

Reply:
@Reviewers feel free to remove if you dont want it

Reply:
It's okay, the main thing is that at the moment the stream works